erpnext/crm/doctype/opportunity/opportunity_dashboard.py

```diff
@@ -20,7 +23,7 @@ def get_data():
 		] + vehicle_domain_links + [
 			{
 				'label': _("Reference"),
-				'items': ['Appointment']
+				'items': ['Appointment', 'Maintenance Schedule']
```

Comment:
Remove Maintenance Schedule link from dashboard... we already have a field for that